src/accordion-item/style.scss

```diff
@@ -0,0 +1,84 @@
+@import "../common/colors.css";
+@import '../common/mediaq.scss';
+
+.wp-block-coliquio-accordion-item {
+    font-family:"Source Sans Pro", sans-serif;
```

Comment:
I think this is not needed, because its inherited from Drupal root CSS.
If you explicit want to define it here, then we should put it into own file because we reuse it in every component.

Reply:
fixed